cyder/management/commands/dhcp_migrate.py

```diff
@@ -98,8 +91,7 @@ def create_subnet(subnet_id, name, subnet, netmask, status, vlan_id):
                       "WHERE vlan_id = %s" % vlan_id):
         vlan_id, vlan_name, vlan_number = cursor.fetchone()
         vlan = Vlan.objects.get(name=vlan_name)
-    vrf = Vrf.objects.get(name=('VoIP' if network_str in voip_networks
-                                else 'Legacy'))
+    vrf = Vrf.objects.get(name='Legacy')
```

Comment:
we won't be running them again, that is correct.

On Fri, Apr 10, 2015 at 3:35 PM, Nathan Murrow <notifications@github.com>
wrote:

> In cyder/management/commands/dhcp_migrate.py
> <https://github.com/OSU-Net/cyder/pull/972#discussion_r28185697>:
>
> > @@ -98,8 +91,7 @@ def create_subnet(subnet_id, name, subnet, netmask, status, vlan_id):
> >                        "WHERE vlan_id = %s" % vlan_id):
> >          vlan_id, vlan_name, vlan_number = cursor.fetchone()
> >          vlan = Vlan.objects.get(name=vlan_name)
> > -    vrf = Vrf.objects.get(name=('VoIP' if network_str in voip_networks
> > -                                else 'Legacy'))
> > +    vrf = Vrf.objects.get(name='Legacy')
>
> Just to be clear... there are no plans to run dhcp_migrate after merging
> this branch, right? So these changes are purely superficial. Is there any
> real reason to maintain dhcp_migrate and dns_migrate at this point? Even
> if we keep them updated, I doubt anyone will be testing them with any
> regularity.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OSU-Net/cyder/pull/972/files#r28185697>.
>